sdl_android/src/main/java/com/smartdevicelink/transport/MultiplexTransportConfig.java

```diff
@@ -35,19 +39,28 @@
 	ComponentName service;
 	int securityLevel;
 
-	
+	List<TransportType> primaryTransports, secondaryTransports;
+	boolean requiresHighBandwidth = false;
```

Comment:
It doesn't seem to me like this should be determined by the app, SDL Core's configuration should be the only determining factor in whether a service is allowed on a given transport (also, I'm fairly certain Core will NAK if an app tries to start video/audio over a transport that isn't list in the StartServiceACK).

Reply:
After writing my other comments, I think I misunderstood the original intention behind the flag. I see that it should be used in conjunction with the information from Core. It just appears that this flag is being improperly used in a few cases (which I have pointed out in other comments).